### ISSUE TITLE
Create handlers for keys, events, timers and tasks using their constructors, not through the global Phoenix-object

### ISSUE THREAD

**Issue by kasper** [kasper/phoenix#109]
Create handlers for keys, events, timers and tasks using their constructors, not through the global Phoenix-object. This should finally separate the distinction between managed and unmanaged handlers and make it obvious why you need to keep a reference.

For this, for the context of the JavaScript scope, shorten the names of handlers as follows:

1. `KeyHandler -> Key`
2. `EventHandler -> Event`
3. `TimerHandler -> Timer`
4. `TaskHandler -> Task`

This will also merge the API for managed and unmanaged handlers. To create a new key handler, you will then then use the following.

```javascript
var handler = new Key('q', [ 'ctrl', 'shift' ], function () {});
```

To use the managed API, you will use the following.

```javascript
Key.on('q', [ 'ctrl', 'shift' ], function () {});
```

**Comment by kasper** [comment]
Implemented in master and will be released in 2.2. I’m happy about this change. 😄

**Comment by mafredri** [comment]
I like it!

![](http://s2.quickmeme.com/img/51/5172847fbb49535e849f907e458b00072cc6ae78f07e6eba6368535af487d4bd.jpg)
